consensus/state.go

```diff
@@ -1326,12 +1327,17 @@ func (cs *State) defaultDecideProposal(height int64, round int32) {
 			}
 		}
 
+		wg := sync.WaitGroup{}
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			for i := 0; i < int(blockParts.Total()); i++ {
+				part := blockParts.GetPart(i)
+				cs.sendInternalMessage(msgInfo{&BlockPartMessage{cs.rs.Height, cs.rs.Round, part}, ""})
+			}
+		}()
```

Comment:
definitely makes sense to put this first!

Do we know if this channel is blocking?

Reply:
its size is 20_000, so I don't think it will block. And if it did, that's really bad